engine/src/flutter/impeller/typographer/backends/skia/typographer_context_skia.cc

```diff
@@ -453,23 +459,24 @@ TypographerContextSkia::CollectNewGlyphs(
       // Rather than computing the bounds at the requested point size and
       // scaling up the bounds, we scale up the font size and request the
       // bounds. This seems to give more accurate bounds information.
-      sk_font.setSize(sk_font.getSize() * scaled_font.scale);
+      sk_font.setSize(sk_font.getSize() *
+                      static_cast<Scalar>(scaled_font.scale));
       sk_font.setSubpixel(true);
 
       for (const auto& glyph_position : run.GetGlyphPositions()) {
-        Point subpixel = TextFrame::ComputeSubpixelPosition(
-            glyph_position, scaled_font.font.GetAxisAlignment(),
-            frame->GetTransform() *
-                Matrix::MakeTranslation(frame->GetOffset()));
+        auto foo =
+            frame->GetTransform() * Matrix::MakeTranslation(frame->GetOffset());
```

Comment:
ooops, I left this debugging code in.  It's going to have to be fixed when I merge with the workaround we are about to land though.